packages/EasyCodingStandard/src/Console/Output/TableOutputFormatter.php

```diff
@@ -36,16 +40,13 @@ public function __construct(
         $this->errorAndDiffCollector = $errorAndDiffCollector;
     }
 
-    public function report(int $processedFilesCount): int
+    public function report(InputInterface $input, OutputInterface $output): int
     {
         $this->reportFileDiffs($this->errorAndDiffCollector->getFileDiffs());
 
         if ($this->errorAndDiffCollector->getErrorCount() === 0
             && $this->errorAndDiffCollector->getFileDiffsCount() === 0
         ) {
-            if ($processedFilesCount) {
-                $this->easyCodingStandardStyle->newLine();
-            }
```

Comment:
Why is this removed? It makes code less readable

Also, the processed file count is needed here

Reply:
My mistake. Again out of scope for this PR.

I was trying to clean up the signature for `report()`. I figured out that if `easyCodingStandardStyle->progressFinish` is used inside Application then this if statement could be removed with `$processedFilesCount`.

Reply:
I see